testsuite/MDAnalysisTests/analysis/test_hole.py

```diff
@@ -74,18 +75,21 @@ def test_HOLE(self, H):
 
         p = profiles_values[0]
 
-        assert len(p) == 425, "wrong number of points in HOLE profile"
+        assert_equal(len(p), 425, err_msg="wrong number of points in HOLE profile")
         assert_almost_equal(p.rxncoord.mean(), -1.41225, err_msg="wrong mean HOLE rxncoord")
         assert_almost_equal(p.radius.min(), 1.19707, err_msg="wrong min HOLE radius")
 
     def test_vmd_surface(self, tmpdir, H):
         with tmpdir.as_cwd():
             filename = H.create_vmd_surface(filename="hole.vmd")
-            assert len(open(filename).readlines()) == 6504, "HOLE VMD surface file is incomplete"
+            with open(filename) as f:
+                nlines = len(f.readlines())
+            assert_equal(nlines, 6504,
```

Comment:
Same as above: why `assert_equal`?

Reply:
Can't remember why I changed it. Should be reverted. Good catch.